src/adapter/graphql/src/scalars/engine_desc.rs

```diff
@@ -39,6 +39,7 @@ impl From<domain::EngineDesc> for EngineDesc {
 
 /////////////////////////////////////////////////////////////////////////////////////////
 
+#[allow(clippy::enum_variant_names)]
```

Comment:
Debatable. Perhaps, we might use `SqlQueryDialect` enum with `Sql` prefix removed from literals. I am unaware of any non-SQL dialects planned

Reply:
Agreed -- after a quick look at the code, I thought it might affect the communication protocols (backend/frontend) and thought it was beyond the scope of my task. 

We can add an issue to the backlog, what do you think?

Reply:
@sergiimk , what do you think re renaming?